common/core/src/main/java/com/vmware/connectors/common/payloads/response/CardAction.java

```diff
@@ -54,6 +54,7 @@
     private boolean removeCardOnCompletion;
 
     @JsonProperty("request")
+    @ToStringExclude
```

Comment:
You can't exclude this.  This is important information for the action.  Sometimes it contains IDs for the business object.  Ex. airwatch's install action: https://github.com/vmware/connectors-workspace-one/blob/120cbca3f9f85d4b34044862e0ca28f5bca9c363/connectors/airwatch/src/main/java/com/vmware/connectors/airwatch/AirWatchController.java#L223

Reply:
Having any``` Set``` and ```Map``` objects makes the ```hash``` value change every time. So I have excluded these fields intentionally.